src/lib/useMessageCache.ts

```diff
@@ -0,0 +1,81 @@
+import { StreamingStepInput } from "./types";
+
+function useMessageCache(
```

Comment:
Why not `export default` here?
```suggestion
export default function useMessageCache(
```

Reply:
yeah will do